### DISCUSSION TITLE
Revocation mechanisms outside of CRL / OSCP

### DISCUSSION THREAD

**Discussion by grahamqs** [strongswan/strongswan#1286] [General]
Hi

Is there any way to deny a peer from connecting to a StrongSwan gateway based on anything but the CRL or OCSP check ?

Cisco have a method called certificate map, where you can apply attributes for the certificate to either match or deny a match. I didn't know if there's anything similar that we can do with strong swan ? Eg. deny based on DN in certificate, certificate serial number, IKE ID etc

many thanks

**Comment by Thermi** [comment]
the extauth plugin
